lib/ex_doc/formatter/html.ex

```diff
@@ -143,9 +144,9 @@ defmodule ExDoc.Formatter.HTML do
       |> Enum.map(&Task.await(&1, :infinity))
 
     api_reference_headers =
-      if(Enum.empty?(modules),    do: [], else: [{"Modules", "modules"}]) ++
-      if(Enum.empty?(exceptions), do: [], else: [{"Exceptions", "exceptions"}]) ++
-      if(Enum.empty?(protocols),  do: [], else: [{"Protocols", "protocols"}])
+      if Enum.empty?(modules),    do: [], else: [{"Modules", "modules"}] ++
```

Comment:
If we want to fix this while complying with Credo, we can write this line and the ones below as:

    (if Enum.empty?(modules),    do: [], else: [{"Modules", "modules"}]) ++

It will maintain the proper precedence and Credo shouldn't warn.